tests/models/grounding_dino/test_modeling_grounding_dino.py

```diff
@@ -115,7 +150,9 @@ def prepare_config_and_inputs(self):
         pixel_values = floats_tensor([self.batch_size, self.num_channels, self.image_size, self.image_size])
         pixel_mask = torch.ones([self.batch_size, self.image_size, self.image_size], device=torch_device)
 
-        input_ids = ids_tensor([self.batch_size, self.max_text_len], self.num_labels)
+        # To avoid erros when running tests with `labels` `input_ids` have to follow this structure
```

Comment:
Following back on this comment - could you add some explanatory detail on what "this structure" means for any future readers of the code?

Reply:
Addressed here https://github.com/huggingface/transformers/pull/31828/commits/9782de07399252abaf734a88a7ef60ceedfacfdf the main thing is that there's a template for the text prompt and based on this structure (separated by a `.`) we derive the `label_maps`, hence the `input_ids` we use for the test should not be random otherwise we'll have issues on the tests that use `labels`